common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -111,15 +111,15 @@
     ${HTML(render_bundle(page))}
 </%def>
 
-<%def name="webpack(entry)">
+<%def name="webpack(entry, extension=None, config='DEFAULT', attrs='')">
     <%doc>
         Loads Javascript onto your page from a Webpack-generated bundle.
         Uses the Django template engine because our webpack loader only provides template tags for Jinja and Django.
     </%doc>
     <%
         body = capture(caller.body)
     %>
-    ${HTML(render_bundle(entry))}
+    ${HTML(render_bundle(entry, extension=None, config='DEFAULT', attrs=attrs))}
```

Comment:
Curious about these additional args -- how do they figure into your long term plan?

Reply:
They are exposed by render_bundle, and I thought I might need them (although thus far, it turns out I didn't). (In particular, I was going to use the `attrs` field for async loading)